### DISCUSSION TITLE
Microsoft Teams/SharePoint autorisations for Draw.io

### DISCUSSION THREAD

**Discussion by bgorrindo** [jgraph/drawio#2987] [Q&A]
Hi,
In Ms Teams, when we try to add or open draw.io doc, we are prompt to set admin autorisation for the first time.
After admin credentials settings, the process loop on admin approval requirement for app.diagrams.net.

If we break the loop and choose "come back to app" , we have this error CAA90019 :
![image](https://user-images.githubusercontent.com/110838449/183448989-ab4cc0c1-8cd6-42e3-9365-36f53c24ece8.png)

How to solve this point or bypass the credential by setting rights in backplane ?
Thank you.

**Comment by davidjgraph** [comment]
We have no access to your Sharepoint. What browser are you using?

**Reply by bgorrindo** [reply]
Google Chrome, Edge Chromium, Firefox (inprivate or not), the problem is the same or almost identical.
We have the impression that the administrator authorization token does not go down once acquired and it loop on Admin Approval.
If we escape, we can see 2 messages very qwickly in Red : Timeout and tockenRevoked .. 

note : We try with Teams App and Teams Online (Edge and Firefox)
